src/app/service/eco-news/create-eco-news.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CreateEcoNewsService {
+
+  constructor() { }
+
+  public translations = {
+    'ua': {
+      text: "",
+      title: ""
+    },
+    'ru': {
+      text: "",
+      title: ""
+    },
+    'en': {
+      text: "",
+      title: ""
+    }
+  }
+
+  public getTranslationByLang(lang: string): {text: string, title: string } {
```

Comment:
Please create appropriate interface for { text: string, title: string }

Reply:
Created